### PR TITLE
Setup dbt locally with BQ on Docker 

### PR DESCRIPTION
**Pull request by ankurchavda**
Raising a PR to add a quick guide to set up dbt on Docker. 

Added three files `docker-compose.yml`, `Dockerfile`  and `docker-setup.md` for ease of access. 
Changed the `README.md` to link it to the setup markdown.

If this is too much clutter then I will just link my repo guide in the `README.md` and remove all other files from here. Let me know :D

### CODE REVIEW

File: week_4_analytics_engineering/README.md [hunk 1]
```diff
@@ -14,8 +14,7 @@ By this stage of the course you should have already:
...
 
-_Optional_: If you feel more comfortable developing locally you could use a local installation of dbt as well. You can follow the [official dbt documentation](https://docs.getdbt.com/dbt-cli/installation) or use a docker image from oficial [dbt repo](https://github.com/dbt-labs/dbt/). You will need to install the latest version (1.0) with the BigQuery adapter (dbt-bigquery).
-After local installation you will have to set up the connection to BQ in the `profiles.yml`, you can find the templates [here](https://docs.getdbt.com/reference/warehouse-profiles/bigquery-profile)
+_Optional_: If you feel more comfortable developing locally you could use a local installation of dbt as well. You can follow the [official dbt documentation](https://docs.getdbt.com/dbt-cli/installation) or follow the [dbt with BigQuery on Docker](docker-setup.md) guide to setup dbt locally on docker.
```
[the remark this answers is not in the document]
Comment (ankurchavda) [reply week_4_analytics_engineering/README.md]: That makes sense. I have added that.

**Comment by alexeygrigorev** [comment]
Maybe we should put it to `docker` folder?

**Comment by ankurchavda** [comment]
Sure I can do that @alexeygrigorev . That would be cleaner as well.

**Comment by ankurchavda** [comment]
@alexeygrigorev @Victoriapm - put the files in `docker_setup folder`

**Comment by alexeygrigorev** [comment]
Is it ready?

**Comment by ankurchavda** [comment]
Yes @alexeygrigorev

**Review by Victoriapm — APPROVED** [review]
looks good to me, thanks! 🚀

**Pull request merged**